modules/acquia_cms_common/tests/src/Functional/SiteStudioElementsPermissionsTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_common\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\user\Entity\Role;
+
+/**
+ * Tests basic, broad permissions of the user roles included with Acquia CMS.
+ *
+ * @group acquia_cms_common
+ * @group acquia_cms
+ * @group risky
+ */
+class SiteStudioElementsPermissionsTest extends BrowserTestBase {
```

Comment:
@rajeshreeputra Since, we are not doing any functional testing, so this could be change to UnitTestCase ?